runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -121,7 +240,7 @@ end
 
 --- Disable inlay hints for a buffer
 ---@param bufnr (integer) Buffer handle, or 0 for current
-local function disable(bufnr)
+local function _disable(bufnr)
```

Comment:
Based on the use of `bufnr` in this function, we should change the type of the parameter to `integer|nil`.

The same applies to `_enable` and `_toggle`.